app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  belongs_to :post
+  belongs_to :author, class_name: 'User'
+
+  after_save :update_post_like_counter
+
```

Comment:
- [ ] you don't have a method name **update_post_like_counter** but you have a method named **likes_counter** so you can put it here :smile: as shown below :+1:

`after_save :likes_counter`